src/Modal/Selection.js

```diff
@@ -248,6 +249,7 @@ export default class Modal_Selection
 
                 inputOptions.push({group: 'Positioning', text: 'Offset selected items position', value: 'offset'});
                 inputOptions.push({group: 'Positioning', text: 'Rotate selected items position', value: 'rotate'});
+                inputOptions.push({group: 'Positioning', text: 'Round selected items\' positions to nearest N meters', value: 'roundtranslation'});
```

Comment:
This should probably be reworded; especially as the units are centimeters not meters.